test/e2e/serial/tests/workload_unschedulable.go

```diff
@@ -340,4 +339,182 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload unsched
+			if len(nrtCandidates) < neededNodes {
+				Skip(fmt.Sprintf("not enough nodes with NUMA zones each of them can match requests: found %d, needed: %d", len(nrtCandidates), neededNodes))
+			}
+
+			// After filter get one of the candidate nodes left
+			nrtCandidateNames := e2enrt.AccumulateNames(nrtCandidates)
+			targetNodeName, ok := nrtCandidateNames.PopAny()
+			Expect(ok).To(BeTrue(), "cannot select a target node among %#v", nrtCandidateNames.List())
+			By(fmt.Sprintf("selecting node to schedule the pod: %q", targetNodeName))
+
+			By("Padding all other candidate nodes")
+			var paddingPods []*corev1.Pod
+			for nodeIdx, nodeName := range nrtCandidateNames.List() {
+
+				nrtInfo, err := e2enrt.FindFromList(nrtCandidates, nodeName)
+				Expect(err).NotTo(HaveOccurred(), "missing NRT info for %q", nodeName)
+
+				baseload, err := nodes.GetLoad(fxt.K8sClient, nodeName)
+				Expect(err).ToNot(HaveOccurred(), "missing node load info for %q", nodeName)
+
+				baseloadResources := corev1.ResourceList{
+					corev1.ResourceCPU:    baseload.CPU,
+					corev1.ResourceMemory: baseload.Memory,
+				}
+
+				paddingResources, err := e2enrt.SaturateNodeUntilLeft(*nrtInfo, baseloadResources)
+				Expect(err).ToNot(HaveOccurred(), "could not get padding resources for node %q", nrtInfo.Name)
+
+				for zoneIdx, zone := range nrtInfo.Zones {
+					podName := fmt.Sprintf("padding%d-%d", nodeIdx, zoneIdx)
+					padPod := newPaddingPod(nodeName, zone.Name, fxt.Namespace.Name, paddingResources[zone.Name])
+
+					padPod, err = pinPodTo(padPod, nodeName, zone.Name)
+					Expect(err).NotTo(HaveOccurred(), "unable to pin pod %q to zone %q", podName, zone.Name)
+
+					err = fxt.Client.Create(context.TODO(), padPod)
+					Expect(err).NotTo(HaveOccurred(), "unable to create pod %q on zone %q", podName, zone.Name)
+
+					paddingPods = append(paddingPods, padPod)
+				}
+			}
+
+			By("Padding target node")
+			//calculate base load on the target node
+			targetNodeBaseLoad, err := nodes.GetLoad(fxt.K8sClient, targetNodeName)
+			Expect(err).ToNot(HaveOccurred(), "missing node load info for %q", targetNodeName)
+
+			// Pad the zones so no one could handle both containers
+			// so we should left enought resources on each zone to accomodate
+			// the "biggest" ( in term of resources) container but not the
+			// sum of both
+			paddingResources := corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("6"),
+				corev1.ResourceMemory: resource.MustParse("6Gi"),
+			}
+
+			targetNrt, err := e2enrt.FindFromList(nrtCandidates, targetNodeName)
+			Expect(err).NotTo(HaveOccurred(), "missing NRT info for targetNode %q", targetNodeName)
+
+			paddingResourcesTargetNodePerZone, err := e2enrt.SaturateNodeUntilLeft(*targetNrt, paddingResources)
+			Expect(err).ToNot(HaveOccurred(), "could not get padding resources for node %q", targetNrt.Name)
+
+			for idx, zone := range targetNrt.Zones {
+				paddingRes := paddingResourcesTargetNodePerZone[zone.Name]
```

Comment:
there actually are multiple valid ways to compute the padding/target resources. And we're still forming our own best practices, so there is a variety showcased in the existing tests. I think the following is clearer and a bit more explicit, but I won't mind alternative (readable :) ) ways as long as they are verified:
```
			By("Padding target node")
			//calculate base load on the target node
			targetNodeBaseLoad, err := nodes.GetLoad(fxt.K8sClient, targetNodeName)
			Expect(err).ToNot(HaveOccurred(), "missing node load info for %q", targetNodeName)

			// Pad the zones so no one could handle both containers
			// so we should left enought resources on each zone to accomodate
			// the "biggest" ( in term of resources) container but not the
			// sum of both
			paddingResources := corev1.ResourceList{
				corev1.ResourceCPU:    // FILLME
				corev1.ResourceMemory: // FILLME
			}

			targetNrt, err := e2enrt.FindFromList(nrtCandidates, targetNodeName)
			Expect(err).NotTo(HaveOccurred(), "missing NRT info for targetNode %q", targetNodeName)

			for idx, zone := range targetNrt.Zones {
				zoneRes := paddingResources.DeepCopy() // extra safety
				if idx == 0 {                          //any zone would do it, we just choose one.
					targetNodeBaseLoad.Apply(zoneRes)
				}

				paddingRes, err := e2enrt.SaturateZoneUntilLeft(zone, zoneRes)
				Expect(err).ToNot(HaveOccurred(), "could not get padding resources for node %q", targetNrt.Name)

				podName := fmt.Sprintf("padding-tgt-%d", targetNodeName, idx)
				padPod := newPaddingPod(targetNodeName, zone.Name, fxt.Namespace.Name, paddingRes)

				padPod, err = pinPodTo(padPod, targetNodeName, zone.Name)
				Expect(err).NotTo(HaveOccurred(), "unable to pin pod %q to zone %q", podName, zone.Name)

				err = fxt.Client.Create(context.TODO(), padPod)
				Expect(err).NotTo(HaveOccurred(), "unable to create pod %q on zone %q", podName, zone.Name)

				paddingPods = append(paddingPods, padPod)
			}
```